apps/e2e/src/client/wsSanity.ts

```diff
@@ -19,7 +23,7 @@ async function wsSanity(context: RunContext) {
     expiration: nowInSeconds() + 60,
     // Limit price
     price: 28000,
-    amount: toFixedPoint(0.01),
+    amount: toFixedPoint(0.01).toString(),
```

Comment:
would it make sense to handle the conversion internally? or have a different version of `toFixedPoint` that returns a string?

Reply:
ah ic, i think this is fine as is 👍